mne/io/tests/test_pick.py

```diff
@@ -8,7 +8,8 @@
 
 from mne import (pick_channels_regexp, pick_types, Epochs,
                  read_forward_solution, rename_channels,
-                 pick_info, pick_channels, __file__, create_info)
+                 pick_info, pick_channels, create_info)
+from mne import __file__ as my_file
```

Comment:
I'm sure that `my_file` does not follow the naming convention. Suggestions are welcome. 

cc:@agramfort @dengemann